SynthSwap-Util/package/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@jaredborders/synthswap-utils",
+  "version": "1.0.0",
+  "description": "synthswap specific utility tools",
+  "main": "index.js",
+  "scripts": {
+    "test": "node test/script.js"
+  },
+  "keywords": [
+    "synthswap"
+  ],
+  "author": "JaredBorders",
+  "license": "ISC",
+  "dependencies": {
+    "abi-decoder": "^2.4.0",
+    "ethers": "^5.6.5"
+  }
+}
```

Comment:
Really was thinking we don't make this a subset of synthswap, but we bundle all of synthswap as an NPM package. Imagine you wanted to integrate synthswap with another project or needed access to the interface. Would be nice just to `npm install @kwenta/synthswap`